docs/data/base/guides/interoperability-with-tailwind-css/interoperability-with-tailwind-css.md

```diff
@@ -0,0 +1,451 @@
++        <Button className="bg-white text-slate-900 dark:bg-slate-100 dark:text-slate-700 flex-none -my-2 mx-auto w-20 h-20 rounded-full border-2 border-slate-900/5 shadow-md flex items-center justify-center" aria-label="Pause">
+           <svg width="30" height="32" fill="currentColor">
+             <rect x="6" y="4" width="4" height="24" rx="2" />
+             <rect x="20" y="4" width="4" height="24" rx="2" />
+           </svg>
+-        </button>
++        </Button>
+         <div className="flex-auto flex items-center justify-evenly">
+-          <button type="button" aria-label="Skip 10 seconds">
++          <Button aria-label="Skip 10 seconds">
+             <svg width="24" height="24" fill="none">
+               <path d="M17.509 16.95c-2.862 2.733-7.501 2.733-10.363 0-2.861-2.734-2.861-7.166 0-9.9 2.862-2.733 7.501-2.733 10.363 0 .38.365.711.759.991 1.176" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+               <path d="M19 5v3.111c0 .491-.398.889-.889.889H15" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+             </svg>
+-          </button>
+-          <button type="button" className="hidden sm:block lg:hidden xl:block" aria-label="Next">
++          </Button>
++          <Button className="hidden sm:block lg:hidden xl:block" aria-label="Next">
+             <svg width="24" height="24" fill="none">
+               <path d="M14 12 6 6v12l8-6Z" fill="currentColor" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+               <path d="M18 6v12" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+             </svg>
+-          </button>
+-          <button type="button" className="rounded-lg text-xs leading-6 font-semibold px-2 ring-2 ring-inset ring-slate-500 text-slate-500 dark:text-slate-100 dark:ring-0 dark:bg-slate-500">
++          </Button>
++          <Button className="rounded-lg text-xs leading-6 font-semibold px-2 border-2 border-slate-500 text-slate-500 dark:text-slate-100 dark:ring-0 dark:bg-slate-500 hover:ring-cyan-500">
+             1x
+-          </button>
++          </Button>
+         </div>
+       </div>
+     </div>
+```
+
+You may noticed that I slightly changed the classes use for the custom borders on some of the buttons so that we can have a consistent focus indicator on all buttons.
+
+## Conclusion
+
+This is what we learned in this guide:
+
+✅ How to apply the Tailwind CSS classes inside the MUI Base components using the `componentsProps` prop, which allows us to target each slot inside the components.\
+✅ We also learned that in some more involved customization scenarios, we may even need to create custom components for some of the slots. We can then use the `components` prop to pass the custom component. When creating custom components, it is important not to forget to pass all props on the root of the component.\
```

Comment:
```suggestion
✅ How to create custom components for specific slots when in more complex customization scenarios.
We used the `component` prop to pass them into the parent component.
```

The "it is important not to forget..." part might be better suited somewhere else.